src/crypto/tls/common.go

```diff
@@ -406,6 +426,10 @@ type ClientHelloInfo struct {
 	// Algorithms Extension is being used (see RFC 5246, Section 7.4.1.4.1).
 	SignatureSchemes []SignatureScheme
 
+	// SignatureSchemesDC lists the signature and hash schemes that the client
+	// is willing to verify when using delegated credentials.
+	SignatureSchemesDC []SignatureScheme
```

Comment:
Can this be nil?

Reply:
Yes, for TLS 1.2 and less. Shall I added in the comment?

Reply:
Yes, would be good.

Reply:
I think `nil` means "use the defaults", like for `SignatureScheme`.

Reply:
Not here. ClientHelloInfo represents the client hello we get, it's not configuration like Signature Scheme. The draft implicitly disallows sending DCs when empty, by saying the chosen signature scheme must be in this list.

Reply:
Added a note.